openml/_api_calls.py

```diff
@@ -161,12 +163,12 @@ def _download_minio_file(
     proxy_client = ProxyManager(proxy) if proxy else None
 
     client = minio.Minio(endpoint=parsed_url.netloc, secure=False, http_client=proxy_client)
-
     try:
         client.fget_object(
             bucket_name=bucket,
             object_name=object_name,
             file_path=str(destination),
+            progress=ProgressBar() if config.show_progress else None,
```

Comment:
I'm not entirely sure how you parse the config file but just ensure it's a boolean as if `config.show_progress == "false"`, this would still evaluate to `True`